### PR TITLE
Campaign endDate update

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request updates the formatting of Campaign endDate in our schema so that it displays properly in our campaign info card.

### How should this be reviewed?

Take a look at the parse function and if it's being used accurately

### Any background context you want to provide?

Trying out using the `date-fns` package to format our endDate similarly to how it's formatted in NS.

### Relevant tickets

References [Pivotal #170090422](https://www.pivotaltracker.com/story/show/170090422).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: src/schema/rogue.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { getFields } from 'fielddataloader';
 import { GraphQLAbsoluteUrl } from 'graphql-url';
 import { GraphQLDateTime } from 'graphql-iso-date';
 import { makeExecutableSchema } from 'graphql-tools';
+import { format as dataFormat, parseISO } from 'date-fns';
```
Comment (DFurnes) [inline src/schema/rogue.js]: If we're going to alias this, can we use `dateFormat` for more clarity?

File: src/schema/rogue.js [hunk 2]
```diff
@@ -619,6 +622,8 @@ const resolvers = {
     actions: (campaign, args, context) =>
       Loader(context).actionsByCampaignId.load(campaign.id),
     causes: campaign => parseCampaignCauses(campaign),
+    // formatting to drop the time stamp in order to correctly display the endDate in Campaign Info
+    endDate: campaign => dataFormat(parseISO(campaign.endDate), 'MM/dd/yyyy'),
```
Comment (DFurnes) [inline src/schema/rogue.js]: Will this handle instances where `campaign.endDate` is null?

File: src/schema/rogue.js (hunk 1 above)
Comment (lindsayrmaher) [reply src/schema/rogue.js]: 🤦‍♀  yah def!

File: src/schema/rogue.js (hunk 2 above)
Comment (lindsayrmaher) [reply src/schema/rogue.js]: updated it to add a check on null endDates!

**Pull request merged**